9.0/Dockerfile

```diff
@@ -131,6 +134,11 @@ RUN set -ex; \
   ln -s /opt/solr/modules /opt/solr/contrib; \
   ln -s /opt/solr/prometheus-exporter /opt/solr/modules/prometheus-exporter;
 
+RUN set -ex; \
+    apt-get update; \
+    apt-get -y install acl dirmngr lsof procps wget netcat gosu tini jattach; \
```

Comment:
This will eventually be auto-generated by `gradle createDockerfileOfficial`. We need this to work with the local dockerfile as well, which will not be installing `dirmngr` above.